### PR TITLE
Fix log level + support auto-fetch of models when label is selected

### PR DESCRIPTION
**Pull request by SachidanandAlle**
1) Fix Log Level
2) Support loading models automatically when user clicks/selects a label
3) Remove progressbar while fetching models (not required)

@YuanTingHsieh @jcfr

### CODE REVIEW

File: slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py [hunk 1]
```diff
@@ -428,7 +428,15 @@ def activate(self):
 
         self.updateGUIFromMRML()
 
+        self.onClickModels()
+        if self.connectedToEditorForSegmentChange is False:
+            self.connectedToEditorForSegmentChange = True
+            editorWidget = slicer.modules.segmenteditor.widgetRepresentation().self()
```
Comment (YuanTingHsieh) [inline slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py]: where is "editorWidget" defined?

File: slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py [hunk 2]
```diff
@@ -194,7 +199,7 @@ def onClickModels(self):
                 self.annotationModelSelector.addItem(model_name)
 
         self.segmentationButton.enabled = self.segmentationModelSelector.count > 0
-        self.updateGUIFromMRML()
+        self.onClickEditPoints()
```
Comment (YuanTingHsieh) [inline slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py]: what is the logic behind this change?

File: slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py (hunk 1 above)
Comment (SachidanandAlle) [reply slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py]: editorWidget = slicer.modules.segmenteditor.widgetRepresentation().self()

self() method returns this object...

File: slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py (hunk 2 above)
Comment (SachidanandAlle) [reply slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py]: whenever user selects a different segment in the listbox.. clear previous and load the stored extreme points for that segment..

**Pull request merged**